guides/common/modules/con_using-pxe-to-provision-hosts.adoc

```diff
@@ -66,3 +66,9 @@ For BIOS provisioning, you must associate a PXELinux template with the operating
 For UEFI provisioning, you must associate a PXEGrub2 template with the operating system.
 If you associate both PXELinux and PXEGrub2 templates, {Project} can deploy configuration files for both on a TFTP server, so that you can switch between PXE loaders easily.
 endif::[]
+
+.Bonded network interfaces
+You can configure a bonded interface on your host and use it for provisioning with PXE boot.
```

Comment:
`use it for provisioning with PXE boot.` -> `use it during PXE based provisioning, after the actual PXE boot.`